### PR TITLE
Switch to yarn from npm

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -157,4 +157,4 @@
     "node": ">= 4.2.1",
     "npm": ">= 3"
   }
-}
+}
```
Comment (kahboom) [inline package.json]: The newline! Please or I'll die from OCD problems.

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 The front end application for Red Hat iPaaS - a flexible, customizable, cloud-hosted platform that provides core integration capabilities as a service. It leverages Red Hat's existing product architecture using OpenShift Online/Dedicated and Fuse Integration Services.
 
-For the middle tier API that this client communicates with, please see [this](https://github.com/redhat-ipaas/ipaas-api) repo. 
+For the middle tier API that this client communicates with, please see [this](https://github.com/redhat-ipaas/ipaas-api) repo.
```
Comment (kahboom) [inline README.md]: I thought I had updated this to reflect the actual path `https://github.com/redhat-ipaas/ipaas-api-java`. @KurtStam said he was fine with keeping the name as is, so I didn't delete the original API repo, but still can if either of you change your mind. Until then, this should be updated.

File: package.json (hunk 1 above)
Comment (jimmidyson) [inline package.json]: Done.

File: README.md (hunk 2 above)
Comment (jimmidyson) [inline README.md]: Done.

Comment (kahboom) [inline README.md]: Wait I think you put `https://github.com/redhat-ipaas/ipaas-api/java` by mistake.

**Comment by jimmidyson** [comment]
BTW did `npm run e2e` work for you before? Seems `yarn e2e` fails for me - think it could be related to some chromedriver issues, but just wanted to check I hadn't broken anything there with this.

**Comment by kahboom** [comment]
@jimmidyson - Nope, you didn't break anything. No need to worry about that as I think I will be fighting it out with QE so that we can maintain E2E tests (at least that's what I'd prefer), but have not set this up yet.

**Comment by kahboom** [comment]
@jimmidyson - Ty! Feel free to merge.

**Pull request merged**